plutus-metatheory/src/Builtin.lagda.md

```diff
@@ -79,7 +95,127 @@ data Builtin : Set where
   mkPairData                      : Builtin
   mkNilData                       : Builtin
   mkNilPairData                   : Builtin
+```
+
+## Signatures
+
+The following module defines a `signature` function assigning to each builtin an abstract type.
+
+```
+private module SugaredSignature where
+```
+Syntactic sugar for writing the signature of built-ins.
+This is defined in its own module so that these definitions are not exported.
+
+```
+    -- number of different type variables
+    ∙ = 0
+    ∀a = 1
+    ∀b,a = 2
```

Comment:
Why not `∀a,b`?

Reply:
Because `a` is the deBruijn index 0, and `b` is 1, so that's the correct order.